src/ui/public/doc_viewer/doc_viewer.js

```diff
@@ -1,47 +1,52 @@
 import _ from 'lodash';
-import angular from 'angular';
-import 'ace';
-import html from 'ui/doc_viewer/doc_viewer.html';
+import $ from 'jquery';
 import 'ui/doc_viewer/doc_viewer.less';
-define(function (require) {
 
+import 'ui/doc_viewer/viewers/table.js';
+import 'ui/doc_viewer/viewers/json.js';
+
+define(function (require) {
 
   require('ui/modules').get('kibana')
   .directive('docViewer', function (config, Private) {
+    const docTableDetailViews = Private(require('ui/registry/doc_table_detail_views'));
```

Comment:
I suggest following the pattern used in https://github.com/elastic/kibana/pull/6102/files. We will be moving to it very soon